test/v1/server/server_test.go

```diff
@@ -29,3 +51,85 @@ func info(t *testing.T) *httpexpect.Response {
 	return e.GET(config.GetBaseURL() + "/info").
 		Expect()
 }
+
+func TestTxForwarder(t *testing.T) {
+	e1 := expectLow(t, config.GetBaseURL())
```

Comment:
Could we just use a single `config.GetBaseURL()` call and dedup the code?

Reply:
Yeah, will optimize it when will work on the next test.